module/Motion/CrossToMid.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file CrossToMid.cpp
+ * @brief サークルの交点から交点へ移動するクラス
+ * @author YKhm20020
+ */
+
+#include "CrossToMid.h"
+
+using namespace std;
+
+CrossToMid::CrossToMid(COLOR _targetColor, double _targetSpeed, int _targetBrightness,
+                       const PidGain& _gain, bool& _isLeftEdge, int _rotateAngle, int _changeAngle)
+  : BlockMotion(1.02, 1.01),
```

Comment:
```suggestion
  : BlockMotion(1.02, 1.01),  // 動作時間, 失敗リスク TODO: 測定し直す
```
みたいなコメントが欲しいです。
他動作も同様